http/src/main/scala/org/http4s/blaze/http/Http1ServerCodec.scala

```diff
@@ -295,7 +298,9 @@ private final class Http1ServerCodec(maxNonBodyBytes: Int, pipeline: TailStage[B
       private var prelude: StringBuilder,
       maxCacheSize: Int) extends InternalWriter {
 
-    prelude.append("transfer-encoding: chunked\r\n\r\n")
+    // The transfer-encoding header will be the last header, and before each write,
+    // we will append a '\r\n{length}\r\n
+    prelude.append("transfer-encoding: chunked\r\n")
```

Comment:
Http2 requires the be transmitted in lower case, so I've been trying to do that consistently. Do you think it's worth changing?